ui/apps/platform/src/Containers/Violations/ViolationsTableSearchFilter.tsx

```diff
@@ -89,8 +92,15 @@ function ViolationsTableSearchFilter({
     onFilterChange,
     onSearch,
 }: ViolationsTableSearchFilterProps) {
+    const { analyticsTrack } = useAnalytics();
+    const trackAppliedFilter = createFilterTracker(analyticsTrack);
     const filterChipGroupDescriptors = makeFilterChipDescriptors(searchFilterConfig);
 
+    function onSearchHandler(payload: OnSearchPayload) {
+        trackAppliedFilter('Policy Violations Filter Applied', payload);
+        onSearch(payload);
```

Comment:
Very small nit: in the unlikely case that our analytics code throws an error, it would be better if the functionality-related code ran first so that it does not impact the user's workflow.